src/layouts/AuthenticatedLayout/index.tsx

```diff
@@ -1,15 +1,17 @@
 import { Outlet } from "react-router-dom";
 import { AuthenticatedHeader } from "./AuthenticatedHeader";
 import { Sidebar } from "./Sidebar";
+import { SidebarProvider } from "@zenml-io/react-component-library";
 
 export function AuthenticatedLayout() {
 	return (
 		<div className="relative flex min-h-screen w-full flex-col">
 			<AuthenticatedHeader />
 			<main className="flex flex-grow flex-col">
 				<div className="flex flex-grow">
-					<Sidebar />
-
+					<SidebarProvider>
```

Comment:
This fixes it for now. Depending on where we need access to the Sidebar we need to move the Provider more up